frontend/catalyst/debug.py

```diff
@@ -68,3 +72,55 @@ def func(x: float):
     else:
         # Dispatch to Python print outside a qjit context.
         builtins.print(x)
+
+
+def get_cmain(fn, *args):
+    """Return a C program that calls a jitted function with the provided arguments.
+
+    Args:
+        fn (QJIT): a qjit-decorated function
+        *args: argument values to use in the C program when invoking ``fn``
+
+    Returns:
+        str: A C program that can be compiled and linked with the current shared object.
```

Comment:
Minor: could we add a small example here? It could be very useful

Reply:
https://github.com/PennyLaneAI/catalyst/pull/529/commits/de6fe29b76c5fe5cd312e82f0698cd440f5e1a53